### PR TITLE
[Task] [68542] Add Handle Event For HomeScreen

### PR DESCRIPTION
**Pull request by bachnd2k1**
### [Redmine link]
https://edu-redmine.sun-asterisk.vn/issues/68542

### CODE REVIEW

File: MovieApp/MovieApp/Controller/Home/HomeViewController.swift [hunk 1]
```diff
@@ -131,3 +139,16 @@ extension HomeViewController: UITableViewDelegate {
         header.textLabel?.text = capitalizedWords.joined(separator: " ")
     }
 }
+
+extension HomeViewController: SectionCellCellDelegate {
+    func showError(error: String) {
+        showAlertError(message: error)
+    }
+    func didTapSectionCell(movie: Movie, youtube: Youtube) {
+        DispatchQueue.main.async { [weak self] in
```
Comment (ghost) [inline MovieApp/MovieApp/Controller/Home/HomeViewController.swift]: chỗ này sao phải dùng Queue Main á

**Review by ghost — APPROVED** [review]

File: MovieApp/MovieApp/Cell/SectionCell.swift [hunk 2]
```diff
@@ -7,6 +7,11 @@
 
 import UIKit
 
+protocol SectionCellCellDelegate : AnyObject {
```
Comment (hlinh88) [inline MovieApp/MovieApp/Cell/SectionCell.swift]:
```suggestion
protocol SectionCellCellDelegate: AnyObject {
```

File: MovieApp/MovieApp/Cell/SectionCell.swift [hunk 3]
```diff
@@ -17,6 +22,7 @@ final class SectionCell: UITableViewCell {
         return collectionView
     }()
     private var movies: [Movie] = [Movie]()
```
Comment (hlinh88) [inline MovieApp/MovieApp/Cell/SectionCell.swift]:
```suggestion
    private var movies = [Movie]()
```

File: MovieApp/MovieApp/Controller/DetailViewController.swift [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+//  DetailViewController.swift
+//  MovieApp
+//
+//  Created by Bach Nghiem on 20/09/2023.
+//
+
+import UIKit
+
+final class DetailViewController: UIViewController {
+
```
Comment (hlinh88) [inline MovieApp/MovieApp/Controller/DetailViewController.swift]:
```suggestion
```

File: MovieApp/MovieApp/Controller/Home/HomeViewController.swift [hunk 5]
```diff
@@ -17,7 +17,6 @@ final class HomeViewController: UIViewController {
     @IBOutlet weak var infoButton: UIButton!
     @IBOutlet weak var scrollView: UIScrollView!
     @IBOutlet weak var tableView: UITableView!
```
Comment (hlinh88) [inline MovieApp/MovieApp/Controller/Home/HomeViewController.swift]: private

File: MovieApp/MovieApp/Controller/Home/HomeViewController.swift [hunk 6]
```diff
@@ -31,6 +30,15 @@ final class HomeViewController: UIViewController {
...
+    @IBAction private func handleListButton(_ sender: Any) {
+    }
+    
+    @IBAction private func handlePlayButton(_ sender: Any) {
+    }
+    
+    @IBAction private func handleInfoButton(_ sender: Any) {
+    }
+    
     func configureHeaderView() {
```
Comment (hlinh88) [inline MovieApp/MovieApp/Controller/Home/HomeViewController.swift]: private

**Review by hlinh88 — APPROVED** [review]

**Pull request merged**